client/index.html

```diff
@@ -218,38 +218,20 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
-									<input type="checkbox" name="join">
-									Show joins
+									<input type="checkbox" name="optional">
+									Show <abbr title="Joins, parts, kicks, nick changes, and mode changes">status messages</abbr>
```

Comment:
Done and mixed with change mentioned in next comment:

<img width="468" alt="screen shot 2017-08-06 at 00 49 39" src="https://user-images.githubusercontent.com/113730/29000947-9894f292-7a49-11e7-869a-065277b99bfa.png">